src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import React from 'react'
+
+const ArticleCard = ({article})=>{
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+    }
+
+    console.log(article.id)
+
+
+    return (
+        <div className="ArticleCard">
```

Comment:
It would be great if we could avoid using divs for accessibility consideration reasons.

Reply:
I'm not sure how to address this further in this situation, I've removed them as far as I feel able..?

Reply:
Take a look at some online resources for some further help if you aren't sure what to implement. This page for example could be useful: https://www.w3schools.com/html/html5_semantic_elements.asp
But there are others.
There is an article tag for example.